addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -670,12 +670,18 @@ def _import_fattura_pa_line(self, element, invoice_line_form, extra_info=None):
         company = invoice_line_form.company_id
         partner = invoice_line_form.partner_id
         message_to_log = []
+        predict_enabled = self.env.context.get('edi_prediction_enabled', False)
```

Comment:
So the idea is using a context key to indicate the prediction functions exist ; essentially telling account_accountant is installed ? :thinking: Quite frankly, I don't like that :/ 

I guess you're doing that to avoid a bridge module ? Then wouldn't it overall just be cleaner to check whether account_accountant is installed ?